Procfile

```diff
@@ -1,3 +1,4 @@
-web: npm start
-clock: npm run start-clock
-worker: npm run start-worker
+clock: IS_HEROKU=true npm run start-clock
+release: IS_HEROKU=true npm run on-release
+web: IS_HEROKU=true npm start
+worker: IS_HEROKU=true npm run start-worker
```

Comment:
Heroku will run the release as long as there's a "release" entry in Procfile, the presence of the env var is so that we can know within our app whether it is running in Heroku or not and whether we need to run migrations etc. ourselves in the start script or not.
By setting the env var here, we don't have to rely on an admin configuring the env var (which was my first attempt at this) or rely on a kludgey hack to check the process.env.NODE variable for the presence of the string "heroku" (which was my 2nd attempt, but I rejected because heroku could decide to change how they populate that env var at any time). By setting it explicitly in the Procfile, it will be reliable that it's true when Refocus is running in Heroku.